src/main/webapp/index-style.css

```diff
@@ -56,4 +56,30 @@
   font-family: arial;
   color: black;
   visibility: hidden;
-}
\ No newline at end of file
+}
+
+
+/* Login Button Specs. */
+.loginButton {
+  height: 30px;
+  width: 30%;
+  position: relative;
+  left: 30%;
```

Comment:
This isn't important, but how do we feel about having the log-in button beneath the searchbar rather than above it?

Reply:
Hm, good suggestion. I don't personally have a preference, which do you think will make it the most accessible for our users?